src/Event/SmsEvents.php

```diff
@@ -27,7 +27,7 @@
   const MESSAGE_GATEWAY = 'sms.message.gateway';
 
   /**
-   * Process and chunk a SMS message before it is queued, sent, or received.
+   * Process a SMS message before it is queued, sent, or received.
```

Comment:
Comment should be more like "Event that is dispatched before an SMS message is queued, sent, or received."